src/AutopilotApi/Client.php

```diff
@@ -293,19 +293,34 @@ public function requestApi(string $path, array $options = []): array
                 ],
                 // @todo Remove "verify" flag post-EA, once service is using trusted cert.
                 'verify' => false,
+                // Do not convert http errors to exceptions
+                'http_errors' => false,
             ],
             $options
         );
 
         $result = $this->request->request($url, $options);
-        if ($result->isError()) {
-            throw new TerminusException(
-                'Failed requesting Autopilot API: {reason}',
-                ['reason' => $result->getStatusCodeReason()]
-            );
+        switch ($result->getStatusCode()) {
+            // Status code in the 200's... good to go
+            case "200":
+                return (array) $result->getData();
+
+            // Status Code in the 409: Conflict
+            case "409":
+                throw new TerminusException('Autopilot already active for that site.');
+            // Status code in the 500's: Some other errors
+            case "500":
+                throw new TerminusException(
+                    'Server Internal Error: %reason',
+                    ['%reason' => $result->getStatusCodeReason() ]
+                );
+
+            default:
+                throw new TerminusException(
```

Comment:
What about another 2xx? Shouldn't this default be only for 4xx and 5xx?

Reply:
My first instinct was to do it by "sections". I should have stuck with that.